src/actions/general.js

```diff
@@ -10,16 +10,25 @@ export function getPing() {
         Client.getPing,
         GeneralTypes.PING_REQUEST,
         GeneralTypes.PING_SUCCESS,
-        GeneralTypes.PING_FAILURE
+        GeneralTypes.PING_FAILURE,
```

Comment:
weird, it didn't issue an error, but either way, i fixed it... I'll take a look at eslint